### ISSUE TITLE
Wake Word Engine Pausing

### ISSUE THREAD

**Issue by rglick64** [alexa/alexa-avs-sample-app#585]
I've had Alexa up and running on a Pi 3 for a few weeks now.  I'm using the 3.5mm audo jack and a Playstation Eye USB microphone, both of which have been working fine.  Suddenly, I've gotten into a situation where when I start the wake word engine, as soon as it detects the wake word, it goes into a paused state.  I don't *think* I've made any changes that would explain this, and I avoid doing software updates because I didn't want to mess with a working config.  See below for the output I'm getting.  NOTE:  I've tried this with both Kitt and Sensory engines with the same result.

INFO:main: Starting Wake Word Agent
INFO:WakeWordAgent: State set to IDLE(2)
INFO:Initializing Sensory library | library name: TrulyHandsfree | library version: 5.0.0-avs.1 | model file: ../ext/resources/spot-alexa-rpi.snsr
WARNING:Library expires on: License expires on 28 May 2017 00:00:00 GMT
INFO:SensoryWakeWordEngine: mainLoop thread started
INFO:WakeWordIPCSocket::mainLoop thread started
INFO:WakeWordIPCSocket: init socket on port:5123
INFO:WakeWordAgent: thread started
INFO:===> Connected to AVS client <===
INFO:===> WakeWordAgent: wake word detected <===
INFO:WakeWordAgent: State set to WAKE_WORD_DETECTED(3)
INFO:WakeWordAgent: State set to SENT_WAKE_WORD_DETECTED(4)
INFO:WakeWordAgent: IPC Command received:3
INFO:WakeWordAgent: State set to WAKE_WORD_PAUSE_REQUESTED(5)
INFO:SensoryWakeWordEngine: handling pause
INFO: *** THREAD JOINING: Sensory ***
INFO:SensoryWakeWordEngine: mainLoop thread ended
INFO:WakeWordAgent: State set to WAKE_WORD_PAUSED(6)

Any ideas what might cause this, or how to resolve?

**Comment by xiaotanzhuying** [comment]
The same with you...
Waiting solutions...

**Comment by dhpp** [comment]
Hello, thanks for this question.  The key item in your log trace is this:

INFO:WakeWordAgent: IPC Command received:3

What this is indicating is that the java sample app is instructing the wake word agent to pause, and release the audio channel.  The java sample app then takes the audio channel, in order to send the user speech to AVS as a SpeechRecognizer.Recognize Event.

The next step is that the java sample app should perform this for the user, then send the wake word agent an unpause IPC message.  It appears this is not being sent.

The next thing I would suggest is you inspect the java client log which you will find under logs/application.log, and compare operations at the time the wake word agent is being paused.  There may be some error output there indicating why the wake word agent is not released.

Let us know what you find in the log, and we can help you investigate further.

**Comment by rglick64** [comment]
Here are the first 200 lines from this log.  note:  I'm getting this issue directly after a reboot and restart of the software using the same method that was working previously.  I think there are a couple of reboots included in these lines.


19:54:47.491 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Finishing downchannel
19:54:47.501 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Establishing downchannel
19:55:02.503 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:55:18.506 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:55:35.509 [DownchannelRequestThread] ERROR com.amazon.alexa.avs.http.AVSClient - There was a problem with the request.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:55:35.511 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Finishing downchannel
19:55:35.512 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Establishing downchannel
19:55:50.513 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:56:06.516 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:56:23.519 [DownchannelRequestThread] ERROR com.amazon.alexa.avs.http.AVSClient - There was a problem with the request.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:56:23.521 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Finishing downchannel
19:56:23.522 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Establishing downchannel
19:56:38.523 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:56:54.526 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:57:11.529 [DownchannelRequestThread] ERROR com.amazon.alexa.avs.http.AVSClient - There was a problem with the request.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:57:11.531 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Finishing downchannel
19:57:11.532 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Establishing downchannel
19:57:26.533 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:57:42.536 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:57:59.539 [DownchannelRequestThread] ERROR com.amazon.alexa.avs.http.AVSClient - There was a problem with the request.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.run(AVSClient.java:499) [classes/:?]
Caused by: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.jetty.InputStreamResponseListener.get(InputStreamResponseListener.java:248) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:275) ~[classes/:?]
	... 8 more
19:57:59.541 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Finishing downchannel
19:57:59.563 [DownchannelRequestThread] INFO  com.amazon.alexa.avs.http.AVSClient - Establishing downchannel
19:58:14.564 [DownchannelRequestThread] WARN  com.amazon.alexa.avs.http.AbstractRetryPolicy - Error occured while making call. This call will retry.
com.amazon.alexa.avs.http.AVSClient$RequestException: java.util.concurrent.TimeoutException
	at com.amazon.alexa.avs.http.AVSClient.doRequestActual(AVSClient.java:280) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$200(AVSClient.java:66) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:229) ~[classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$2.call(AVSClient.java:224) ~[classes/:?]
	at com.amazon.alexa.avs.http.AbstractRetryPolicy.tryCall(AbstractRetryPolicy.java:38) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.doRequest(AVSClient.java:235) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient.access$400(AVSClient.java:66) [classes/:?]
	at com.amazon.alexa.avs.http.AVSClient$DownchannelRequestThread.openConnection(AVSClient.java:535) [classes/:?]

**Comment by bcobra** [comment]
This is also happening for me

**Comment by elstryan** [comment]
Hi Folks, 
Based on this stack trace it looks like your client is unable to establish a connection with AVS. If your Java version was updated then perhaps you need to update the ALPN version. Can you please verify that you are using the right ALPN version as per the sub step 3 on this link: https://github.com/alexa/alexa-avs-sample-app/wiki/Linux#6---run-the-sample-app 

Also this is probably a silly question but just in case: can you both please verify that your PIs are connected to the internet?

**Comment by rglick64** [comment]
Fixed!  Elstryan, always ask the silly question!  Your question about being connected to the Internet jogged my memory.  I haven't made any changes on the Pi recently.  However, I did run into an issue recently where my router was being flaky in terms of providing DNS services.  I changed the DHCP service in the router to point directly at my ISP's DNS servers instead of using the router itself as a relay to them.  However, since the Pi isn't using DNS, it still had the router set up in /etc/resolv.conf as the nameserver.  Once I put my ISP's DNS servers in /etc/resolv.conf and rebooted, everything is working again.

Thanks so much for your help!!
